collectors/python.d.plugin/mysql/README.md

```diff
@@ -5,8 +5,19 @@ Module monitors one or more mysql servers
 **Requirements:**
 
 -   python library [MySQLdb](https://github.com/PyMySQL/mysqlclient-python) (faster) or [PyMySQL](https://github.com/PyMySQL/PyMySQL) (slower)
+-   `netdata` local user to connect to the MySQL server. 
 
-It will produce following charts (if data is available):
+To create the `netdata` user, execute the following:
```

Comment:
It seems obvious but still worth to mention - execute in the mysql shell.

Reply:
Definitely worth mentioning! Good catch.

Reply:
Yes, since the `netdata` user is already created upon installation, it only makes sense to note that this `netdata` user is to be created in the MySQL shell. 

I have added it. 

Thanks @ilyam8